community/modules/scripts/ramble-setup/main.tf

```diff
@@ -72,7 +72,7 @@ locals {
     "destination" = "install_ramble.yml"
   }
 
-  bucket_md5  = substr(md5("${var.project_id}.${var.deployment_name}"), 0, 4)
+  bucket_md5  = substr(md5("${var.project_id}.${var.deployment_name}"), 0, 6)
```

Comment:
nit: I thought consensus was to go longer. I would say double it (8). @mr0re1 wanted even longer.

Reply:
Done